morf-h2/src/main/java/org/alfasoftware/morf/jdbc/h2/H2.java

```diff
@@ -141,4 +149,36 @@ public boolean matchesProduct(String product) {
   public Optional<JdbcUrlElements> extractJdbcUrl(String url) {
     return Optional.empty();
   }
+
+  
+  @Override
+  public Optional<UrlConnectionResourcesBean> urlToConnectionResources(String url) {
```

Comment:
This is a utility method, right? It doesn't @Override something. Should it be static?

Reply:
It does override the method in `DatabaseType`.